### DISCUSSION TITLE
Sending compressed image through the network

### DISCUSSION THREAD

**Discussion by plinioMoreno** [robotology/community#141] [Q&A] [answered]
In the case you are in a network where the only way you can communicate to the robot is wi-fi, is there a way to send compressed images in YARP (in the case of ROS is the [compressed image transport plugin](http://wiki.ros.org/compressed_image_transport)) in a port?

A compressed protocol would useful to visualize the images while keeping the low bandwitdth constraint of wi-fi.

**Comment by randaz81** [comment, marked as answer]
hi, you can check the yarp mjpeg carrier
`yarp connect /a /b mjpeg`

**Comment by plinioMoreno** [comment]
Thanks @randaz81, I'll try out that and let you know.
